toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -556,6 +556,10 @@ build_rpm_in_chroot_no_install pyproject-rpm-macros
 chroot_and_install_rpms pyproject-rpm-macros pyproject-rpm-macros
 chroot_and_install_rpms pyproject-rpm-macros pyproject-srpm-macros
 
+# ocaml and other ocmal packages require ocaml-srpm-macros 
+build_rpm_in_chroot_no_install ocaml-srpm-macros
+chroot_and_install_rpms ocaml-srpm-macros ocaml-srpm-macros
```

Comment:
Since we're not building any OCAML packages in the toolchain, we might as well drop this one line:
```suggestion
```

The package will still get built and included in the toolchain and chroot.